app/execution.js

```diff
@@ -417,15 +417,15 @@ exports.loadDataAndSendNotification = function loadDataAndSendNotification(repor
                             {
                                 channel: channels[index],
                                 notificationSent: false,
-                                status: "Something wrong while creating an image"
+                                status: "Visualisation disabled due to configuration changes. Please reschedule the report to activate."
```

Comment:
check the message with @admin-vizcentric venkat

Reply:
its already confirm by Venkat